lib/core/utils.js

```diff
@@ -109,6 +109,17 @@ RESUtils.isMatchURL = function(moduleID) {
 
 	return included;
 };
+RESUtils.minifyOptions = function(obj) {
+	var min = {};
+	if (obj) {
+		for (var key in obj) {
+			if ('value' in obj[key]) {
+				min[key] = {value: obj[key].value};
+			}
+		}
+	}
+	return min;
+};
```

Comment:
I think value(s) is all that needs to go into storage. Lemme check through to see if any `RESStorage.getItem('RESoptions...'` happen outside the usual "load data from storage and merge into schema / default values" routine.